app/controllers/presentations_controller.rb

```diff
@@ -1,8 +1,10 @@
 class PresentationsController < ApplicationController
   def index
-    @presentations = Presentation.joins(:event)
-                                 .merge(Event.order(date: :desc))
-                                 .page(params[:page])
+    @q = Presentation.ransack(params[:q])
```

Comment:
I'd like this instance variable named a little bit more intuitively. I am guessing it means query but I don't know if that's what it is.